examples/streaming-pwa/sw.js

```diff
@@ -30,11 +30,14 @@ const makeBookStream = request => {
     // On start,
     start(controller) {
 
-      // This function processes a stream coming in (from a Response object)
-      const processStream = response => {
+      // This function processes a ReadableStream (from Response.body)
+      const processStream = body => {
+
+        // no Response.body stream available
+        if (!body) return controller.close()
```

Comment:
Without this images don't load for me. Version 78.0.3904.70 (Official Build) (64-bit)
 on MacOS Mojave 10.14.5 (18F132)